driver/web/docs/resources/admin/data-content-items.yaml

```diff
@@ -0,0 +1,84 @@
+post:
+  tags:
+    - Admin 
+  summary: Admin API that Creates a data content item
+  description: |
+    Creates a new data content item
+
+    **Auth:** Requires admin token with `all_admin_content` permission
+  security:
+    - bearerAuth: [] 
+  requestBody:
+     description: Creates a data content item
+     content:
+       application/json:
+         schema:
+           $ref: "../../schemas/apis/admin/data-content-item/DataContentItem.yaml"          
+  responses:
+    200:
+      description: Success
```

Comment:
The docs should show that you are returning the created content item in your handler.